bcda/client/bluebutton_test.go

```diff
@@ -516,6 +534,7 @@ func (s *BBRequestTestSuite) TestValidateRequest() {
 				noExcludeSAMHSAChecker,
 				noIncludeAddressFieldsChecker,
 				noIncludeTaxNumbersChecker,
+				hasDefaultRequestHeaders,
```

Comment:
For this scenario, can you also add an assertion method that confirms the bulk request headers are not present? (instead of leaving the assertion off for hasBulkRequestHeaders) Since this call makes an underlying call to the getRawData.

Reply:
Updated

Reply:
https://github.com/CMSgov/bcda-app/pull/812/files#diff-3642e6e64fc5fe565b5224862f2f624cba8e76e0ebc297dd0c778d98b0fa2d89R922